### DISCUSSION TITLE
https://www.dcm4che.org/maven2 down

### DISCUSSION THREAD

**Discussion by lcl45** [dcm4che/dcm4che#1047] [Q&A] [answered]
Hi, it appears that https://www.dcm4che.org/maven2 is not reachable. Also the https://maven.scijava.org/content/repositories/public/ repo seems to have only some versions, from the 2.0.21 to 2.0.29. I need the last version available. You know of some other available (and hopefully working) repos? Thanks!!

**Comment by maltempi** [comment]
Same issue here

**Comment by gunterze** [comment, marked as answer]
outage resolved
